server/api/cvrs.py

```diff
@@ -1261,6 +1297,7 @@ def parse_cvrs():
             session,
         )
         session.commit()
+        clean_up_file_system()
```

Comment:
Manually inspected my `/tmp` directory to verify that this cleanup did in fact work as expected